jobbergate-cli/jobbergate_cli/subapps/applications/tools.py

```diff
@@ -185,6 +190,39 @@ def upload_application(
         return response_code == 201
 
 
+def download_application_files(
```

Comment:
This function would be better named `save_application_files`

Reply:
solved in [802a6e0](https://github.com/omnivector-solutions/jobbergate/pull/198/commits/802a6e0dbdd5c472b04832387af5b93e73949eb9)